README.md

```diff
@@ -106,17 +106,6 @@ To organize and steer the development of
 - Discord: [##java-operator-sdk](https://discord.com/invite/K4YEJ3Z5Mw)
 - [Mailing list][of-mailing-list] (Subscribe to access the agenda document.)
 
-## Helm Operator Plugins Working Group
-
-To organize and steer the development of [helm-operator-plugins](https://github.com/operator-framework/helm-operator-plugins) and related projects.
-
-### Meetings
-
-* Issue Triage and Community Meeting: Tuesdays (biweekly) at 11:00 Eastern Time. [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Eastern%20Time%29).
-    * [Meeting Link][meeting-link]: Password: 77777
-    * [Issue Query](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc)
-    * [Agenda](https://docs.google.com/document/d/112b0ZhSid8exnm72sMT7rDzBEkLlE5s74s0pcn44ssg/edit#)
-
 ### Organizers
 
 * Fabian von Feilitzsch (**[@fabianvf](https://github.com/fabianvf)**), Red Hat
```

Comment:
Looks like the `Organizers` and `Contact` sections should also be removed.

Did we check with Simon about this?

Reply:
Updated. No one was coming to the meetings for months. We even changed them to monthly.